src/header-parser.ts

```diff
@@ -321,15 +317,13 @@ export function formatPreamble(
     }
   }
   if (description != null) {
-    // @ts-ignore childNodes is iterable
-    const isJustElements = [...(description.childNodes as Iterable<Node>)].every(
+    const isJustElements = Array.prototype.every.call(
```

Comment:
Can you revert this change? As a matter of style I prefer spreading and calling array methods directly over using `.call`.

Reply:
That's forfeiting the possibility of not exhausting the whole list if every short-circuits; but okay, it wouldn't really buy anything here, anyway.